runtime/caml/memprof.h

```diff
@@ -28,7 +28,9 @@
 
 enum { CAML_MEMPROF_SRC_NORMAL = 0,
        CAML_MEMPROF_SRC_MARSHAL = 1, /* interning */
-       CAML_MEMPROF_SRC_CUSTOM = 2 /* custom memory */ };
+       CAML_MEMPROF_SRC_CUSTOM = 2, /* custom memory */
+       CAML_MEMPROF_SRC_MAP_FILE = 3, /* Unix.map_file */
```

Comment:
```suggestion
       CAML_MEMPROF_SRC_MAP_FILE = 3, /* mmapped file */
```
No reason for being specific to Unix.map_file